components/PostDetails/index.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
```

Comment:
Why would you import all React modules?

Maybe this is easier 
```suggestion
import React from "react";
```

Or even better... 

```suggestion
// Do not import React altogether!
```

👀

Reply:
Ugh, that was an automatic completion for Fragment component, sorry for that!